tools/dissector/packet-opendds.cpp

```diff
@@ -546,7 +546,7 @@ namespace OpenDDS
           tvb_,
           offset,
           (gint) header.message_length_,
-          "No Topic Found for %s \n",
+          "No Topic Found for %C \n",
```

Comment:
This was changed along with the ACE_DEBUG statements, but it's not an ACE_DEBUG (also on line 583)